### ISSUE TITLE
Unable to resolve module 'scheduler/tracing' 

### ISSUE THREAD

**Issue by liemdo** [facebook/react-native#21783]
- [x ] Review the documentation: https://facebook.github.io/react-native
  - [x ] Search for existing issues: https://github.com/facebook/react-native/issues
  - [x ] Use the latest React Native release: https://github.com/facebook/react-native/releases

## Environment
React Native Environment Info:
    System:
      OS: macOS High Sierra 10.13.6
      CPU: x64 Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
      Memory: 73.24 MB / 8.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 9.11.1 - /usr/local/bin/node
      npm: 6.4.1 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.0, macOS 10.14, tvOS 12.0, watchOS 5.0
    IDEs:
      Android Studio: 3.0 AI-171.4443003
      Xcode: 10.0/10A255 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.6.0-alpha.0 => 16.6.0-alpha.0 
      react-native: 0.57.3 => 0.57.3 
    npmGlobalPackages:
      create-react-native-app: 1.0.0
      react-native-cli: 2.0.1

## Description
After I upgraded to 0.57.3, I get this error after running `react-native ios`

```bash
error: bundling failed: Error: Unable to resolve module `scheduler/tracing` from `node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js`: Module `scheduler/tracing` does not exist in the Haste module map
```

## Reproducible Demo
Let us know how to reproduce the issue. Include a code sample, share a project, or     share an app that reproduces the issue using https://snack.expo.io/. Please follow the guidelines for providing a MCVE: https://stackoverflow.com/help/mcve

**Comment by aminosman** [comment]
Looks like an issue with RM `0.57.3` as issue went away after downgrading to `0.57.2`

**Comment by smoothdvd** [comment]
> `rm -rf node_modules && npm install`
> 
> Anyway when there is a dependency error, it is npm-related. And if it ever worked with the same package.json, it can probably be resolved using this command.

Not working

**Comment by quynhnguyen68** [comment]
I'm trying to fix this problem by following this instruction https://github.com/facebook/react-native/issues/21150. 
I  have used yarn add schedule@0.4.0 or schedule@0.5.0 but it doesn't work :(

**Comment by smoothdvd** [comment]
@quynhnguyen68 I fix the issue follow this link [https://github.com/facebook/react-native/issues/21150#issuecomment-429305215](url)

Change react version to "16.6.0-alpha.8af6728"

**Comment by quynhnguyen68** [comment]
Yeah, it works well when I change react to version to "16.6.0-alpha.8af6728"
